mlrun/projects/operations.py

```diff
@@ -187,6 +187,11 @@ def my_pipe(url=""):
     task.spec.verbose = task.spec.verbose or verbose
 
     if engine == "kfp":
+        if schedule:
+            raise mlrun.errors.MLRunInvalidArgumentError(
+                "Scheduling is not supported in KFP run function."
+                "Use either project.set_schedule() or project.run() to schedule runs."
+            )
```

Comment:
"Scheduling job is not supported when running a workflow with kfp engine."
"Use project.run_function()."

Reply:
on second thought this is better - 
"Scheduling job is not supported when running a workflow with kfp engine."